html/changelogs/DreamySkrell-coc-badlands-weight.yml

```diff
@@ -0,0 +1,7 @@
+
+author: DreamySkrell
+
+delete-after: True
+
+changes:
+  - rscadd: "Lower Badlands spawn weight of some CoC ships."
```

Comment:
You're not adding anything new really, this should be more like, the `balance` tag

Reply:
done